data/scripts.js

```diff
@@ -3592,7 +3594,25 @@ exports.BattleScripts = {
 			}
 
 			let monotypeRandomBattleSets = {
+				// Electric
```

Comment:
IMO, applying to both sets is fine.
This is random battle, so we shouldn't care all that much of the full team. It would be a different story if this were Battle Factory.